### ISSUE TITLE
[Bug]: Unable to use custom Okta URL

### ISSUE THREAD

**Issue by jmeekinsint** [Snowflake-Labs/terraform-provider-snowflake#3104]
### Terraform CLI Version

1.7.3

### Terraform Provider Version

0.95.0

### Terraform Configuration

```terraform
provider "snowflake" {
  user               = "svc_account" # required if not using profile or token. Can also be set via SNOWFLAKE_USER env var
  password               = "svc_account_password"
  authenticator     = "Okta"
  okta_url = "https://myapps.custom.com"
  role      = "APP_ADMIN"
  account = "xxxxxx.us-east-1" # Specify the Snowflake account and region
}
```

### Category

category:provider_config

### Expected Behavior

Ability to authentication using service account

### Actual Behavior

Planning failed. Terraform encountered an error while generating this plan.

╷
│ Error: open snowflake connection: 260011: failed to parse an authenticator: https://myapps.custom.com
│ 
│   with provider["registry.terraform.io/snowflake-labs/snowflake"],
│   on main.tf line 24, in provider "snowflake":
│   24: provider "snowflake" {
│ 
╵

### Steps to Reproduce

Copy terraform provider information from problem. 

Run terraform plan

IF you change url to have *.okta.com you get a different error - The specified authenticator is not accepted by your Snowflake account configuration.  Please contact your local system administrator to get the correct URL to use.

### How much impact is this issue causing?

Medium

### Would you like to implement a fix?

- [ ] Yeah, I'll take it 😎

**Comment by sfc-gh-jmichalak** [comment]
Hi @jmeekinsint 👋 
Okta URL host needs to to have a suffix `okta.com` (from the driver documentation):
> To authenticate through Okta, specify https://<okta_account_name>.okta.com (URL prefix for Okta).

We are currently reworking the provider configuration, and we will adjust our documentation to be more descriptive.
About the second error, have you followed the instructions at https://docs.snowflake.com/en/user-guide/oauth-okta? It looks like a misconfiguration on your side, because this error comes from Snowflake.

**Comment by sfc-gh-jmichalak** [comment]
Hi @chriskl  👋

We've released a new v0.98.0 version ([release](https://github.com/Snowflake-Labs/terraform-provider-snowflake/releases/tag/v0.98.0), [migration guide](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/MIGRATION_GUIDE.md#v0970--v0980)) with updated configuration documentation.

**Comment by sfc-gh-jmichalak** [comment]
Closing due to inactivity. Please create a new issue if you have more suggestions.
